server/config/config.go

```diff
@@ -1022,7 +1024,8 @@ func (man Manager) addConfigs() {
 	man.addConfigString("lambda.audit_function", "",
 		"Lambda function name for audit logs")
 
-	// S3 for file carving
+	// S3 for file carving: Deprecated
+	// TODO(JVE): do we need to add a deprecation warning to all these messages?
```

Comment:
that's a good idea if easily doable! can you also hide them please? so people don't find them